grants_tagger/create_inclusion_list.py

```diff
@@ -0,0 +1,66 @@
+"""
+Script to automatically create a list of MeSH terms to include
+(i.e. filtering out the ones that have an attribute which mentions "Do not use")
+given a link to the MeSH xml file
+"""
+import typer
+
+import os
+import httpx
+import uuid
+import xml.etree.ElementTree as ET  # this is a library that can parse xml easily
+import pandas as pd
+from pathlib import Path
+
+
+def inclusion(xml_path, out_path):
+
```

Comment:
Could you also add a docstring to the function? I can see you have a nice explanatory comment at the top, but it would be good to specifically explain your function as well.

Reply:
Great, thanks 🙂 love a nice docstring
I like adding the type in brackets to the arguments to make it even less ambiguous when you have lots of arguments, but pretty obvious here anyway... (I think Justin added the exact formatting to the data science portal)